src/metabase/related.clj

```diff
@@ -0,0 +1,249 @@
+(ns metabase.related
```

Comment:
Does this maybe make more sense as `metabase.models.related` since it has to do with `models` stuff?